tools/manymounts.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# can be used for testing rc_unmount:
+#    # ./tools/manymounts.sh
+#    # rc_unmount -- -p '^/tmp/manymounts.*'
```

Comment:
Not sure if this should be kept or not. Included it so that others can test easily. Can remove if desired.